config/config.yaml

```
@@ -59,27 +63,27 @@ scheduling:
+  #   Trả lời theo định dạng:
+  #   SUMMARY: [tóm tắt bằng tiếng Việt]
+  #   KEYWORDS: [từ khóa 1], [từ khóa 2], [từ khóa 3], [từ khóa 4], [từ khóa 5]
 
 # AI search categories configuration (new feature)
 ai_search_categories:
   ai-news:
-    enabled: true
+    enabled: false
     category: "Artificial Intelligence"
     search_prompt: |
       Search for the latest 5 news articles about artificial intelligence, machine learning, AI research, and AI applications published today. 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)